tests/testthat/test-utils-format.R

```diff
@@ -0,0 +1,18 @@
+context("utils-format")
+
+test_that("pluralise works correctly", {
+  expect_identical(pluralise("[an] index(es)", c("x")), "an index")
+  expect_identical(pluralise("[an] index(es)", c("x", "y")), "indexes")
+})
+
+test_that("pluralise leaves alone parentheses / square brackets that have spaces inside", {
+  expect_identical(pluralise("[an] index(es )", c("x")), "an index(es )")
+  expect_identical(pluralise("[an ] index(es)", c("x", "y")), "[an ] indexes")
+})
```

Comment:
These special characters may be part of the error message.

What does `pluralise("[an ] index(es )", c("x"))` return with your implementation?